cpp/ql/lib/semmle/code/cpp/Variable.qll

```diff
@@ -136,6 +136,9 @@ class Variable extends Declaration, @variable {
   /**
    * Gets an assignment expression that assigns to this variable.
    * For example: `x=...` or `x+=...`.
+   *
+   * This does _not_ include the initialization of the variable. Use
+   * `Variable.getInitializer()` to get the variable's initializer.
```

Comment:
We could mention `Variable.getAnAssignedValue()` here (or if we make a new class like Java has, that).

Reply:
Good point. I've added a mention of `Variable.getAnAssignedValue()` in 586c5b90c156781a8f9c1f13e4c87ef5d7c1ec70.

Reply:
I would add a clarification in `Variable.getAnAssignedValue()` that the returned value includes both `Assignment`s and `Initializer`s as the name indicates otherwise?..

Reply:
I've done this now in 207dcb08a788bb400cdeb34f6e5174b5edfa8002. I didn't do it initially because I couldn't find a sentence structure that I was happy with. Do you think it's sufficiently clear now?